test/test_listen_handler.py

```diff
@@ -68,22 +68,24 @@ def test_listen_endpoint(mock_get_current_user,
 
 
 def test_listen_endpoint_not_found(mock_get_current_user,
```

Comment:
We can add another assert statement to endpoint `test_listen_endpoint_not_found`:
```
assert 'detail' in response.json()
```